### DISCUSSION TITLE
The quality is extremely poor

### DISCUSSION THREAD

**Discussion by otrumb** [KRTirtho/spotube#1173] [General]
Nice project, nice idea! Thank you!

But the quality is extremely poor, have you scaled the quality up, right?
Is there any way to improve the quality?

**Comment by Bwangster12** [comment]
I'm not an audiophile, but I'm not in alignment with this.  It sounds as good (if not better) than Spotify.

**Comment by KRTirtho** [comment]
> Nice project, nice idea! Thank you!
> 
> But the quality is extremely poor, have you scaled the quality up, right?
> Is there any way to improve the quality?

You can use Jiosaavn as the audio source from "Settings" > "Audio Source"

It has 320kbps mp4 support. But it has a smaller collection of songs so it might fall back to YouTube when it can't find
